app/services/store_sp_metadata_in_session.rb

```diff
@@ -82,12 +44,12 @@ def update_session
       request_url: sp_request.url,
       request_id: sp_request.uuid,
       requested_attributes: sp_request.requested_attributes,
-      ial: ial_value,
-      ial2: ial2_value,
-      ialmax: ialmax_value,
-      aal_level_requested: aal_level_requested_value,
-      piv_cac_requested: piv_cac_requested_value,
-      phishing_resistant_requested: phishing_resistant_value,
+      ial: parsed_vot&.ial_value_requested,
+      ial2: parsed_vot&.ial2_requested?,
+      ialmax: parsed_vot&.ialmax_requested?,
+      aal_level_requested: parsed_vot&.aal_level_requested,
+      piv_cac_requested: parsed_vot&.piv_cac_requested?,
+      phishing_resistant_requested: parsed_vot&.phishing_resistant?,
```

Comment:
I wouldn't change anything here. These will be individually removed in the sibling PR to stop writiing values which will be merged after the stop reading PRs are deployed. See #10126, #10128, #10130 or #10140 for examples.

Reply:
Removed.